### PR TITLE
[App V2] User Testing Feedback pt1

### PR DESCRIPTION
**Pull request by biyeun**
@orangejenny 
cc @benrudolph 

fixes:
- tablet overlap of buttons
- overlap of app preview in formbuilder
- syntax error on form settings page
- terrible error where the whole page automatically gets hidden on formbuilders's `onunload` event capture, rendering it useless.

adds:
- friendly fix me buttons to build errors directing users to what to do next
- reduce time between transitions
- cleans up some styling here and there

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html [hunk 1]
```diff
@@ -165,7 +165,7 @@ <h4 class="panel-title panel-title-nolink">
                     <td class="col-sm-5">
                         <div data-bind="template: {
                             name: 'case-config:case-properties:property',
-                            data: {property: $data, suggestedProperties: case_transaction.suggestedSaveProperties}
+                            data: {property: $data, suggestedProperties: case_transaction.suggestedSaveProperties},
```
Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html]: haha oops

File: corehq/apps/style/static/app_manager/less/new_appmanager/content.less [hunk 2]
```diff
@@ -21,6 +21,13 @@
   &.formdesigner-content-wrapper {
     padding: 0;
   }
+  #formdesigner {
+    background-color: @cc-brand-low !important;
+    .fd-content-right .fd-head,
+    .fd-content-right .fd-form-actions {
+      margin-right: -2px;
```
Comment (benrudolph) [inline corehq/apps/style/static/app_manager/less/new_appmanager/content.less]: sometimes these type of css styles confuse me when i'm looking to make a change because i'm not sure what they are trying to achieve (usually it's subtle). maybe a comment here?

File: corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html [hunk 3]
```diff
@@ -12,9 +12,9 @@ <h4 class="alert-heading">
                     <span>
                         {% case error.type "blank form" %}
                             {% if not error.message %}
-                                <i class="fa fa-plus"></i> <strong>Add Question</strong> to the
+                                <strong>Add a Question</strong> to the
```
Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: nit trans (for all of this file)

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/style/static/app_manager/less/new_appmanager/content.less (hunk 2 above)
Comment (biyeun) [inline corehq/apps/style/static/app_manager/less/new_appmanager/content.less]: There is a 2px white line between the right edge of form player and the right edge of the screen. This gets rid of it. If I make the container 2px bigger, all of formplayer bumps down to the next line

File: corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html (hunk 3 above)
Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: I wasn't sure if there was a reason the build errors were not translated @orangejenny

File: corehq/apps/style/static/app_manager/less/new_appmanager/content.less (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/style/static/app_manager/less/new_appmanager/content.less]: agree that would make a good comment

File: corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html (hunk 3 above)
Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: I think it's just an oversight & translations would be good.

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: ok, actually pretty non trivial to add trans here.

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: I'm going to leave that for a different PR

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html]: by non trivial...I mean to do with properly we'd have to `{%with foo as bar %}` a lot of the variables. and restructure the if statements, otherwise we'll get a lot of loosely contexted translations

File: corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html (hunk 1 above)
Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html]: was this you, ben? LOL

Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html]: haha no just thought it was funny

Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html]: at least i have no recollection of doing it
